.travis.yml

```diff
@@ -11,8 +11,10 @@ matrix:
     # Restore matrix job entry from d2d67fab to test OSX
     - python: 2.7
       env: UNSUPPORTED=true
+      script: .misc/check_unsupported.sh
     - python: 3.3
       env: UNSUPPORTED=true
```

Comment:
do we still need those? Or shall we keep it so we can see it inside the build overview?